### PR TITLE
Add an example helm values yaml for Vault integration user guide

### PR DESCRIPTION
**Pull request by lei-tang**
Add an example helm values yaml for Vault integration user guide.
This example helm values yaml is referred in https://github.com/istio/istio.io/pull/3098.

### CODE REVIEW

**Comment by lei-tang** [comment]
https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values-istio-sds-auth.yaml has been used. Please see https://github.com/istio/istio.io/pull/3098. The example deployment file in this PR is for user reference.

**Comment by quanjielin** [comment]
I mean add something like https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values-istio-sds-auth.yaml and put all vault config in it, user don't need to have nodeagent deployed in each of their application pod. Just suggestion, up to you.

**Comment by lei-tang** [comment]
@quanjielin Done, added as a helm values yaml.

File: install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    udsPath: "unix:/var/run/sds/uds_path"
+    useNormalJwt: true
+
+nodeagent:
+  enabled: true
+  image: node-agent-k8s
+  env:
+    CA_ADDR: "https://35.233.249.249:8200"
+    CA_PROVIDER: "VaultCA"
+    VAULT_ADDR: "https://35.233.249.249:8200"
```
Comment (mandarjog) [inline install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml]: does this have to be a real ip?

File: install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    VAULT_ROLE: "istio-cert"
+    VAULT_SIGN_CSR_PATH: "istio_ca/sign/istio-pki-role"
+    VAULT_TLS_ROOT_CERT: '-----BEGIN CERTIFICATE-----\nMIIC3jCCAcagAwIBAgIRAIcSFH1jneS0XPz5r2QDbigwDQYJKoZIhvcNAQELBQAw\nEDEOMAwGA1UEChMFVmF1bHQwIBcNMTgxMjI2MDkwMDU3WhgPMjExODEyMDIwOTAw\nNTdaMBAxDjAMBgNVBAoTBVZhdWx0MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIB\nCgKCAQEA2q5lfJCLAOTEjX3xV8qMLEX8zUQpd0AjD6zzOMzx51GVM7Plf7CJmaDq\nyloRz3zcrTEltHUrln5fvouvp4TetOlqEU979vvccnFLgXrSpn+Zt/EyjE0rUYY3\n5e2qxy9bP2E7zJSKONIT6zRDd2zUQGH3zUem1ZG0GFY1ZL5qFSOIy+PvuQ4u8HCa\n1CcnHmI613fVDbFbaxuF2G2MIwCZ/Fg6KBd9kgU7uCOvkbR4AtRe0ntwweIjOIas\nFiohPQzVY4obrYZiTV43HT4lGti7ySn2c96UnRSnmHLWyBb7cafd4WZN/t+OmYSd\nooxCVQ2Zqub6NlZ5OySYOz/0BJq6DQIDAQABozEwLzAOBgNVHQ8BAf8EBAMCBaAw\nDAYDVR0TAQH/BAIwADAPBgNVHREECDAGhwQj6fn5MA0GCSqGSIb3DQEBCwUAA4IB\nAQBORvUcW0wgg/Wo1aKFaZQuPPFVLjOZat0QpCJYNDhsSIO4Y0JS+Y1cEIkvXB3S\nQ3D7IfNP0gh1fhtP/d45LQSPqpyJF5vKWAvwa/LSPKpw2+Zys4oDahcH+SEKiQco\nIhkkHNEgC4LEKEaGvY4A8Cw7uWWquUJB16AapSSnkeD2vTcxErfCO59yR7yEWDa6\n8j6QNzmGNj2YXtT86+Mmedhfh65Rrh94mhAPQHBAdCNGCUwZ6zHPQ6Z1rj+x3Wm9\ngqpveVq2olloNbnLNmM3V6F9mqSZACgADmRqf42bixeHczkTfRDKThJcpY5U44vy\nw4Nm32yDWhD6AC68rDkXX68m\n-----END CERTIFICATE-----'
```
Comment (mandarjog) [inline install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml]: can you use multi line yaml format here?

File: install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml (hunk 1 above)
Comment (lei-tang) [reply install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml]: This has to be the IP of the testing Vault server.

File: install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml (hunk 2 above)
Comment (lei-tang) [reply install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml]: Tried multiline yaml format but it does not work.

File: install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml (hunk 1 above)
Comment (lei-tang) [reply install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml]: Added a comment for the IP address.

**Pull request merged**